packages/ref-metrics-example/pyproject.toml

```diff
@@ -27,13 +27,13 @@ dependencies = [
     "dask>=2024.10.0",
 ]
 
+[project.license]
+text = "Apache-2.0"
+
 [tool.uv]
 dev-dependencies = [
 ]
 
-[tool.uv.sources]
```

Comment:
This look unrelated, did you delete it accidentally?

Reply:
I took the chance to remove an extra block while I was in the file. It isn't needed as the root project already defines it.